components/Home.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Appbar, IconButton, Text, Subheading, Menu } from 'react-native-paper'
+import { useAuthenticationContext } from '../context/authentication'
+
+import theme from '../default-theme'
+
+const UI_STRINGS = {
+  default: {
+    heading: `You don't have a secret set up yet.`,
+    description:
+      'Add one by scanning or uploading a QR code, or even enter the details manually.',
+  },
+}
+
+const A11Y_LABELS = {
+  code: 'Add new secret by scanning a QR Code',
+  upload: 'Add new secret by uploading a QR Code',
+  add: 'Add new secret by filling the details',
+}
+
+export default function Home() {
+  const [isMenuActive, setMenuActive] = React.useState(false)
+  const { handleLogout } = useAuthenticationContext()
+  return (
```

Comment:
formatting: let's put a blank line between the variable definition and the return value